skyfield/units.py

```diff
@@ -187,6 +190,8 @@ def __getattr__(self, name):
         raise AttributeError('no attribute named %r' % (name,))
 
     def __str__(self):
+        if self.radians.size == 0:
+            return '0 values'
```

Comment:
Having a zero `0` in the string might make folks confused, since it looks like a quantity, and `0 values` won't communicate to them what kind of object they've printed out. How could we make the output string include the type, so that, like the normal str and repr, this communicates what kind of value this is? `Empty Angle` or something like that?

Reply:
My intent here was to match the existing way all of the `str`'s and `repr`'s work. I didn't notice that this is the only case that doesn't include units implying the object type. I think something like `<Angle [empty]>`, or `<Angle []>` would be fine here.

Reply:
I like `<Angle []>`, great idea! Try pivoting to that pattern, I think it will look good and communicate clearly.